backend/open_webui/socket/utils.py

```
@@ -11,23 +11,34 @@ def __init__(self, redis_url, lock_name, timeout_secs):
-        # nx=True will only set this key if it _hasn't_ already been set
-        self.lock_obtained = self.redis.set(
-            self.lock_name, self.lock_id, nx=True, ex=self.timeout_secs
-        )
-        return self.lock_obtained
+    def acquire_lock(self):
+        try:
+            # nx=True will only set this key if it _hasn't_ already been set
+            self.lock_obtained = self.redis.set(
+                self.lock_name, self.lock_id, nx=True, ex=self.timeout_secs
+            )
+            return self.lock_obtained
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)